cms/djangoapps/contentstore/course_info_model.py

```diff
@@ -165,7 +167,27 @@ def _get_html(course_updates_items):
     Method to create course_updates_html from course_updates items
     """
     list_items = []
-    for update in reversed(course_updates_items):
+    date_items = []
+    no_date_items = []
+
+    #print course_updates_items
+
+    #for a in (course_updates_items):
+    #   print a.get("date")
+
+
+    for item in course_updates_items:
+        try:
+            parser.parse(item.get("date"))
+            date_items.append(item)
+        except ValueError:
+            no_date_items.append(item)
```

Comment:
Shouldn't you continue to iterate `for item in reverse(course_updates_items)` for backwards compatibility?